src/routes.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
-use Julienbourdeau\RouteUsage\Http\Controllers\RouteUsageController;
+use Illuminate\Support\Facades\Route;
 
-Route::get('route-usage', [RouteUsageController::class, 'index'])->name('route-usage.index');
+Route::get('/', 'RouteUsageController@index')->name('route-usage.index');
```

Comment:
I definitely prefer keeping the route declaration.
Typically like what Freek shows here https://freek.dev/1324-simplifying-controllers

Reply:
This code is based on Laravel Telescope library @julienbourdeau 
```
// One for the service provider
https://github.com/laravel/telescope/blob/2.0/src/TelescopeServiceProvider.php#L27

// And one for the routes
https://github.com/laravel/telescope/blob/2.0/src/Http/routes.php

Reply:
Yes I understand but for now there is only one route and I think it makes it less clear (because namespace and middleware are hidden. It makes a lot of sense when you register many routes, but in this case I find it a bit overkill.